### PR TITLE
Fixes #911

### LINKED ISSUE #911: HCB Climate Directory: Badge tooltips
https://hackclub.com/hcb/climate/

Hook up the badge tooltips to display the badge's name on hover. At the moment, it's hard coded to "Recommended".

image

### PR DESCRIPTION
**Pull request by mjh316**
Fixed the tooltips sharing CSS content property through .tooltip by appending the badge ID as a suffix to the `.tooltipped` class in order to have unique `content` CSS values for each badge.

Fixes issue #911

### CODE REVIEW

**Comment by lachlanjc** [comment]
#### 📝 Changed routes:


| [**/hcb/climate**](https://site-63k5kgd1p.hackclub.dev/hcb/climate) |
|:-:|
|<a href="https://site-63k5kgd1p.hackclub.dev/hcb/climate">Screenshot of /hcb/climate</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/6ocs4Xg577dTP-9BvxV4dT0BTReRw0qIZcgWFKz9yRUFkQ5Dhal26cUAzdMFZTjdNreC40tqpdlhj5HEDlRaNQ.png">(view full size)</a> |







Commit 43db131636ce7cd34acf851dc4c01dbdc04864fe (https://site-63k5kgd1p.hackclub.dev).

**Review by sampoder — APPROVED** [review]
awesome!

**Review by sampoder — COMMENTED** [review]
Thoughts on using https://github.com/vercel/styled-jsx instead?

File: pages/hcb/climate/index.js [hunk 1]
```diff
@@ -671,13 +671,16 @@ export default function ClimatePage({ rawOrganizations, pageRegion }) {
...
 
-                {badges.forOrg(modalOrganization).map((badge, i) => (
-                  <Tooltip.N key={i} text={badge.tooltip}>
-                    <span class="tooltipped">
-                      <Badge badge={badge} />
-                    </span>
-                  </Tooltip.N>
-                ))}
+                {badges.forOrg(modalOrganization).map((badge, i) => {
+                  console.log('badge', badge.label)
```
Comment (sampoder) [inline pages/hcb/climate/index.js]:
```suggestion
```

**Comment by sampoder** [comment]
LGTM

**Comment by sampoder** [comment]
(I tried JSX-ing, no luck: #922)

**Pull request merged**